app/src/main/java/org/mozilla/fenix/settings/logins/fragment/AddNewLoginFragment.kt

```diff
@@ -0,0 +1,87 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.settings.logins.fragment
+
+import android.os.Bundle
+import android.view.Menu
+import android.view.MenuInflater
+import android.view.MenuItem
+import android.view.View
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.lifecycleScope
+import androidx.navigation.fragment.findNavController
+import mozilla.components.support.ktx.android.view.hideKeyboard
+import org.mozilla.fenix.R
+import org.mozilla.fenix.components.StoreProvider
+import org.mozilla.fenix.databinding.FragmentAddNewLoginBinding
+import org.mozilla.fenix.ext.components
+import org.mozilla.fenix.ext.settings
+import org.mozilla.fenix.ext.showToolbar
+import org.mozilla.fenix.ext.toEditable
+import org.mozilla.fenix.settings.logins.LoginsFragmentStore
+import org.mozilla.fenix.settings.logins.controller.SavedLoginsStorageController
+import org.mozilla.fenix.settings.logins.createInitialLoginsListState
+import org.mozilla.fenix.settings.logins.interactor.EditLoginInteractor
+
+/**
+ * Displays the editable new login information for a single website
+ */
+class AddNewLoginFragment : Fragment(R.layout.fragment_add_new_login) {
```

Comment:
I wonder if there's a naming convention we should follow here. Take a look to see if credit cards have "add new" yet and see what they named it

Reply:
Sure!
Well, there's no special class for adding new card but it's the same for editing and adding.
There's no "new" word, just `onAddCreditCardClick()` or `handleAddCreditCardClicked()`
Should it sound similar way?